packages/gotrue/lib/src/types/user.dart

```diff
@@ -268,3 +268,12 @@ class UserIdentity {
         updatedAt.hashCode;
   }
 }
+
+class ResendResponse {
+  /// Only set for phone resend
+  String? messageId;
+
+  ResendResponse({
+    this.messageId,
+  });
+}
```

Comment:
Just a small thing, but maybe this should be in `auth_response.dart` file?

Reply:
Oh sure, fixed.